src/test/scala/uk/gov/hmrc/test/ui/pages/common/ConfirmationPage.scala

```diff
@@ -92,16 +92,33 @@ object ConfirmationPage extends BasePage {
   private def checkPageWhenReceived(): Unit = {
     println("\n========== Version of the Confirmation page: RECEIVED ==========")
     val mrn = checkTable()
-    checkContentLinks(List(
-      linkToTimeline, linkToClearanceHub, linkToMovements, linkToMovements, linkToTimeline, linkToSfus(mrn), linkToFeedback, linkToSupport
-    ))
+    checkContentLinks(
+      List(
+        linkToTimeline,
+        linkToClearanceHub,
+        linkToMovements,
+        linkToMovements,
+        linkToTimeline,
+        linkToSfus(mrn),
+        linkToFeedback,
+        linkToSupport
+      )
+    )
   }
 
   private def checkPageWhenAccepted(): Unit = {
     println("\n========== Version of the Confirmation page: ACCEPTED ==========")
```

Comment:
Remove this println

Reply:
At this point I would remove all `println`